### PR TITLE
Add Region Geochart

### PR DESCRIPTION
**Pull request by andreeanica16**

### CODE REVIEW

**Comment by andreeanica16** [comment]
@tblanshard @juliocc @iht

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -18,6 +18,7 @@
       <a href="#sdkAnchor">SDK Analysis</a>
       <a href="#resourceAnchor">Resource Analysis</a>
       <a href="#locationAnchor">Location Analysis</a>
+      <a href="#regionGeochart" id="hiddenLink"hidden>Region Geochart</a>
```
Comment (tblanshard) [inline src/main/webapp/index.html:21]: Tiny change
```suggestion
      <a href="#regionGeochart" id="hiddenLink" hidden>Region Geochart</a>
```

**Review by tblanshard — DISMISSED** [review]
LGTM :)

**Review by tblanshard — APPROVED** [review]
LGTM :)

**Pull request merged**